igvc_gazebo/nodes/sim_color_detector/sim_color_detector.cpp

```diff
@@ -109,68 +118,9 @@ void handleImage(const sensor_msgs::ImageConstPtr& msg, const sensor_msgs::Camer
   g_pubs.at(camera_name).barrel.publish(outmsg);
```

Comment:
It doesn't seem like we actually publish the barrel segmentation here, just a blank matrix. We should probably add a TODO for that. Actually, it's probably fine that we don't segment the barrels since it would add unnecessary work and we don't really do anything with the barrel segmentation.